botorch/test_functions/multi_objective_fidelity.py

```diff
@@ -0,0 +1,157 @@
+r"""
+Multi-objective optimization benchmark problems.
+
+References
+
+.. [Irshad2021]
+    Irshad, Faran, Stefan Karsch, and Andreas Döpp.
+    Expected hypervolume improvement for simultaneous multi-objective and
+    multi-fidelity optimization. arXiv preprint arXiv:2112.13901 (2021).
+"""
+
+import math
+
+import torch
+from botorch.test_functions.base import MultiObjectiveTestProblem
+from torch import Tensor
+
+
+class MOMFBraninCurrin(MultiObjectiveTestProblem):
+    r"""Augmented Branin-Currin test function for multi-objective
```

Comment:
Let's make this first sentence a short description that fits on one line (for the autodocs)